### DISCUSSION TITLE
How to implement "select all/none" option in multi-select?

### DISCUSSION THREAD

**Discussion by vincerubinetti** [chakra-ui/zag#1055] [Q&A]
What do you think the appropriate way is to do something like this:

<img width="165" alt="Screenshot 2023-11-27 at 7 50 59 PM" src="https://github.com/chakra-ui/zag/assets/8326331/53bbc227-1353-4c85-9d25-6857ace53f12">

I could put a button element above the other options, but then keyboard-only users would not be able to access it. Maybe a "select/deselect" all functionality should be built-in?

**Comment by segunadebayo** [comment]
Closing as outdated — please refer to the updated v1.0 documentation for current API patterns.
